README.md

```diff
@@ -191,6 +191,20 @@ See also:
 * [Example Queries](examples/README.md)
 * [Connect to Firebolt Core](https://docs.firebolt.io/FireboltCore/firebolt-core-connect.html)
 
+## Firebolt Core UI
+
+1. Download the Firebolt Core UI Docker image:
+```bash
+docker pull 231290928314.dkr.ecr.us-east-1.amazonaws.com/firebolt-core-ui:latest
+```
+
+2. Run the UI container, linking it to your Firebolt Core instance:
+```bash
+docker run --name firebolt-core-ui -p 9100:9100 -e FIREBOLT_CORE_URL=http://host.docker.internal:3473 231290928314.dkr.ecr.us-east-1.amazonaws.com/firebolt-core-ui
```

Comment:
Perhaps we need here a suggestion mentioning that `host.docker.internal` should be replaced with any of the node hosts/IPs? `host.docker.internal` is a Mac-specific example, AFAIK